apps/core/scenes/results/Results.js

```diff
@@ -4,6 +4,8 @@ import * as React from 'react';
 import { View } from 'react-native';
 import { QueryRenderer, graphql } from '@kiwicom/margarita-relay';
 import { StyleSheet, Text } from '@kiwicom/universal-components';
+import { DateTime } from 'luxon';
```

Comment:
As mentioned in @JoseDuSV 's PR, we'll need to get rid of `luxon`; we will need to stop PRs using `luxon` after this one is landed.

Reply:
OK